<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>935</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>153</red>
        <green>153</green>
        <blue>153</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>153</red>
        <green>153</green>
        <blue>153</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>153</red>
        <green>153</green>
        <blue>153</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MVVM Bank Client</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>512</width>
       <height>923</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>512</width>
       <height>923</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="passportID_frame">
        <layout class="QVBoxLayout" name="passport_section">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit_passport">
           <property name="maximumSize">
            <size>
             <width>496</width>
             <height>32</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>129</red>
                 <green>130</green>
                 <blue>133</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>129</red>
                 <green>130</green>
                 <blue>133</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>129</red>
                 <green>130</green>
                 <blue>133</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="maxLength">
            <number>32764</number>
           </property>
           <property name="placeholderText">
            <string>Passport id</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_passport">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>* Sequence of numbers without any other symbols. 
 Example: 4874241622</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_surname">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>Surname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_lastName">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>Last name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="sex_frame">
        <layout class="QVBoxLayout" name="sex_section">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_female">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Female</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_male">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Male</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="dob_frame">
        <layout class="QHBoxLayout" name="dob_section">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QDateEdit" name="dateEdit_dob">
           <property name="maximumSize">
            <size>
             <width>260</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_dob">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Date of birth</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_placeOfBirth">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>Place of birth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_country">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>Country</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="dateOfIssue_frame">
        <layout class="QHBoxLayout" name="dateOfIssue_section">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QDateEdit" name="dateEdit_dateOfIssue">
           <property name="maximumSize">
            <size>
             <width>260</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label__dateOfIssue">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Date of the passport issue</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="dateOfExpire_frame">
        <layout class="QHBoxLayout" name="dateOfExpire_section">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QDateEdit" name="dateEdit_dateOfExpire">
           <property name="maximumSize">
            <size>
             <width>260</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_dateOfExpire">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Date of the passport expire</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_placeOfIssue">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>Passport place of issue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_passportAuthority">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>Passport authority</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_adressOfLivivng">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>Address of living</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_email">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>129</red>
              <green>130</green>
              <blue>133</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="placeholderText">
         <string>email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="exit_frame">
        <layout class="QHBoxLayout" name="exitFromRegForm_section">
         <item>
          <widget class="QPushButton" name="pushButton_cancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_signUp">
           <property name="text">
            <string>Sign up</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
